docs/products/kafka/concepts/kafka-tiered-storage.md

```diff
@@ -27,58 +23,96 @@ control over data storage needs.
 
 Tiered storage offers multiple benefits, including:
 
--   **Scalability:** With tiered storage in Aiven for Apache Kafka,
-    storage and computing are effectively decoupled, enabling them to
-    scale independently. This flexibility ensures that while the storage
-    capacity can expand almost infinitely with cloud solutions, compute
-    resources can also be adjusted based on demand, thereby eliminating any
-    concerns about storage or processing limitations.
--   **Cost efficiency:** By moving less frequently accessed data to a
-    cost-effective storage tier, you can achieve significant financial
-    savings.
--   **Operational speed:** With the bulk of data offloaded to remote
-    storage, service rebalancing in Aiven for Apache Kafka becomes
-    faster, making for a smoother operational experience.
--   **Infinite data retention:** With the scalability of cloud storage,
-    you can achieve unlimited data retention, valuable for analytics and
-    compliance.
--   **Transparency:** Even older Kafka clients can benefit from tiered
-    storage without needing to be explicitly aware of it.
-
-## When to and why use it
+- **Scalability:** Tiered storage in Aiven for Apache Kafka decouples storage and
+  computing, allowing them to scale independently. Storage capacity can expand almost
+  infinitely with cloud solutions, and compute resources can adjust based on demand,
+  eliminating concerns about storage or processing limitations.
+- **Cost efficiency:** By moving less frequently accessed data to a
+  cost-effective storage tier, you can achieve significant financial
+  savings.
+- **Operational speed:** With the bulk of data offloaded to remote
+  storage, service rebalancing in Aiven for Apache Kafka becomes
+  faster, making for a smoother operational experience.
+- **Infinite data retention:** Offloading bulk data to remote storage speeds up service
+  rebalancing in Aiven for Apache Kafka, enhancing operational efficiency.
+- **Transparency:** Even older Kafka clients can benefit from tiered
```

Comment:
By "clients" do you mean users?
Also how about `without needing to be explicitly aware of it` > `even unaware of it`?

Reply:
By "clients," I was referring to the Kafka clients, which are applications or services that produce or consume messages from Kafka topics. Most Kafka users are familiar with what clients refer to.